src/services/goToDefinition.ts

```diff
@@ -274,8 +274,9 @@ function getDefinitionFromObjectLiteralElement(typeChecker: TypeChecker, node: N
     if (element) {
         const contextualType = element && typeChecker.getContextualType(element.parent);
         if (contextualType) {
-            return flatMap(getPropertySymbolsFromContextualType(element, typeChecker, contextualType, /*unionSymbolOk*/ false), propertySymbol =>
+            const definitions = flatMap(getPropertySymbolsFromContextualType(element, typeChecker, contextualType, /*unionSymbolOk*/ false), propertySymbol =>

```

Comment:
This seems okay, though, I half wonder if it's better to change this to always return an empty array, then fix the one caller who cares about `undefined` to check length instead. Maybe Andrew has further comments here.